content/en/building/guides/integrations/fhir.md

```diff
@@ -0,0 +1,407 @@
+          "short": {
+            "en": "Patient Name"
+          }
+        },
+        "position": 1,
+        "type": "string",
+        "length": [
+          1,
+          40
+        ],
+        "required": true
+      },
+      "phone_number": {
+        "labels": {
+          "short": {
+            "en": "patient Phone"
+          }
+        },
+        "position": 2,
+        "flags":{
+          "allow_duplicate": false
+        },
+        "type": "phone_number",
+        "required": true
+      },
+      "location_id": {
+        "labels": {
+          "short": {
+            "en": "location_id"
+          }
+        },
+        "position": 3,
+        "type": "string",
+        "length": [
+          1,
+          60
+        ],
+        "required": true
+      },
+      "external_id": {
+        "labels": {
+          "short": {
+            "en": "External ID"
+          }
+        },
+        "position": 4,
+        "type": "string",
+        "length": [
+          1,
+          60
+        ],
+        "required": true
+      }
+    },
+    "public_form": true
+  }
+}
+```
+
+Using the above example form, this transition will create a patient document and add a `birth_date` field using `patient_age_in_days`.
+
+```json
+{
+  "form": "interop_patient",
+  "events": [
+    {
+      "name": "on_create",
+      "trigger": "add_birth_date",
+      "params": "",
+      "bool_expr": "doc.patient_name"
+    },
+    {
+      "name": "on_create",
+      "trigger": "add_patient",
+      "params": {
+        "parent_id": "location_id"
+      },
+      "bool_expr": "doc.patient_name"
+    }
+  ]
+}
+```
+
+## Inbound Reports
+
+[Encounters](https://build.fhir.org/encounter.html) and [Observations](https://build.fhir.org/observation.html) created by interoperating systems can be sent to the CHT to be visible to CHT users.
+Similarly to patients, a mediator converts the FHIR resources to a json format that is submitted to the records API.
+Reports need to be linked to patients using a `patient_id` field which is the uuid of the patient document in CHT. The mediator extracts this id from the `CHT Document ID` identifier of the fhir patient. 
+For patients created by CHT, they need to have been sent to the interoperating system before receiving any reports. For patients created by the interoperating system, the `CHT Document ID` needs to have been set; see the section below on [Patient Ids](building/guides/integrations/fhir/#populating-ids).
+
+A CHT form needs to be configured to receive the reports via the records API.
+In the form configuration, the names of fields which should be extracted from [Observations](https://build.fhir.org/observation.html) should be the codes of the [Observations](https://build.fhir.org/observation.html). Human readable labels can be added for display.
+
+Because these forms are CHT forms like an other, messages, validations and other transitions can be assigned to it to build more complex workflows.
```

Comment:
```suggestion
Because these forms are CHT forms like an other, messages, validations and other transitions can be assigned to them to build more complex workflows.
```